backoffice/src/Helpers/SearchHelpers/DroitQueryBuilder.cs

```diff
@@ -99,7 +99,13 @@ public static IQueryable<Droit> BuildQuery(DroitSearchForm form, IQueryable<Droi
                   form.StatusList.Contains(d.Status) ));
             query = query.Where(d =>
                 ( form.TriageNumbers.IsNullOrEmpty() ||
-                  (d.TriageNumber!= null && form.TriageNumbers.Contains(d.TriageNumber.Value) )));
+                  ((d.TriageNumber == null && form.TriageNumbers.Contains(0)) ||
```

Comment:
Zero is basically a non-triage number they can select for ones without a Triage number set